# Bulgarian translation for furiusisomount
# Copyright (c) 2010 Rosetta Contributors and Canonical Ltd 2010
# This file is distributed under the same license as the furiusisomount package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2010.
#
msgid ""
msgstr ""
"Project-Id-Version: furiusisomount\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2010-08-25 11:45+0100\n"
"PO-Revision-Date: 2010-11-15 18:45+0000\n"
"Last-Translator: Svetoslav Stefanov <svetlisashkov@yahoo.com>\n"
"Language-Team: Bulgarian <bg@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2013-03-21 04:48+0000\n"
"X-Generator: Launchpad (build 16532)\n"

#: aboutbox.py:35
msgid "translator-credits"
msgstr ""
"Launchpad Contributions:\n"
"  Svetoslav Stefanov https://launchpad.net/~svetlisashkov"

#: aboutbox.py:40
msgid "Developer:"
msgstr "Разработчик:"

#: filehash.py:25 filehash.py:36 main.py:246
msgid "Calculating, please wait..."
msgstr "Изчислява се, моля изчакайте..."

#: globals.py:22
msgid "Furius ISO Mount Tool"
msgstr "Инструмент Furius ISO Mount"

#: globals.py:23
msgid ""
"Simple Gtk+ Interface to Mount ISO, IMG, BIN, MDF and NRG Image files "
"without burning to disk."
msgstr ""
"Прост Gtk+ интерфейс за монтиране на ISO, IMG, BIN, MDF и NRG образи без да "
"се записват на диск."

#: imageaction.py:35 imageaction.py:58 imageaction.py:69 imageaction.py:79
#: main.py:83 main.py:340 main.py:386
#, python-format
msgid ""
"Error mounting image.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при монтиране на образа.\n"
"Неочаквана грешка: %s"

#: imageaction.py:36
msgid "Defaulting to home directory for mounting."
msgstr "Връщане към домашната папка за монтиране."

#: imageaction.py:55
#, python-format
msgid ""
"%s successfully mounted @ %s. If required, run the following commands to "
"remove:\n"
"  :fusermount -u %s\n"
"  :rmdir %s"
msgstr ""
"%s е монтиран успешно в %s. Ако е необходимо, изпълнете следните команди за "
"премахване:\n"
"  :fusermount -u %s\n"
"  :rmdir %s"

#: imageaction.py:66
#, python-format
msgid ""
"%s successfully mounted @ %s. If required, run the following commands to "
"remove:\n"
"  :gksu -u root umount %s\n"
"  :rmdir %s"
msgstr ""
"%s е монтиран успешно в %s. Ако е необходимо, изпълнете следните команди за "
"премахване:\n"
"  :gksu -u root umount %s\n"
"  :rmdir %s"

#: imageaction.py:76 main.py:79 main.py:336 main.py:382
#, python-format
msgid ""
"Error mounting image.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при монтиране на образа.\n"
"Грешка на операционната система(%s): %s"

#: imageaction.py:103
#, python-format
msgid ""
"Error saving history to file.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при запазване на историята във файл.\n"
"Грешка на операционната система(%s): %s"

#: imageaction.py:106
#, python-format
msgid ""
"Error saving history to file\n"
"Unexpected error: %s"
msgstr ""
"Грешка при запазване на историята във файл.\n"
"Неочаквана грешка: %s"

#: imageaction.py:128
#, python-format
msgid "%s successfully unmounted."
msgstr "%s е демонтиран успешно"

#: imageaction.py:130 main.py:358 main.py:401
#, python-format
msgid ""
"Error unmounting image.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при демонтиране на образа.\n"
"Неочаквана грешка: %s"

#: imageaction.py:142 main.py:364
#, python-format
msgid ""
"Error burning image.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при записване на образа.\n"
"Неочаквана грешка: %s"

#: imageaction.py:152
#, python-format
msgid ""
"Error launching nautilus.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при стартиране на nautilus.\n"
"Неочаквана грешка: %s"

#: main.py:51
msgid "Application started..."
msgstr "Приложението е стартирано..."

#: main.py:81 main.py:338 main.py:384
#, python-format
msgid ""
"Error mounting image.\n"
"IO error(%s): %s"
msgstr ""
"Грешка при монтиране на образа.\n"
"Входно-изходна грешка(%s): %s"

#: main.py:95 main.py:112
#, python-format
msgid ""
"Error loading history.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при зареждане на историята\n"
"Грешка на операционната система(%s): %s"

#: main.py:97 main.py:114
#, python-format
msgid ""
"Error loading history.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при зареждане на историята\n"
"Неочаквана грешка: %s"

#: main.py:125
#, python-format
msgid ""
"Error creating settings directory.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при създаване на папката с настройки.\n"
"Грешка на операционната система(%s): %s"

#: main.py:127
#, python-format
msgid ""
"Error creating settings directory.\n"
"IO error(%s): %s"
msgstr ""
"Грешка при създаване на папката с настройки.\n"
"Входно-изходна грешка(%s): %s"

#: main.py:129
#, python-format
msgid ""
"Error creating settings directory.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при създаване на папката с настройки.\n"
"Неочаквана грешка: %s"

#: main.py:139
#, python-format
msgid "New settings file created at %s"
msgstr "В %s е създаден нов файл с настройки"

#: main.py:141
#, python-format
msgid ""
"Error creating settings file.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при създаване на файла с настройки.\n"
"Грешка на операционната система(%s): %s"

#: main.py:143
#, python-format
msgid ""
"Error creating settings file.\n"
"IO error(%s): %s"
msgstr ""
"Грешка при създаване на файла с настройки.\n"
"Входно-изходна грешка(%s): %s"

#: main.py:145
#, python-format
msgid ""
"Error creating settings file.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при създаване на файла с настройки.\n"
"Неочаквана грешка: %s"

#: main.py:159
msgid "_Browse..."
msgstr "_Разглеждане..."

#: main.py:163
msgid "_Mount"
msgstr "_Монтиране"

#: main.py:167 main.py:234 main.py:260
msgid "_Checksum"
msgstr "_Контролна сума"

#: main.py:171
msgid "Bur_n"
msgstr "_Запис"

#: main.py:175
msgid "_Unmount"
msgstr "_Демонтиране"

#: main.py:176
msgid "View Log"
msgstr "Преглед на дневника"

#: main.py:177
msgid "Delete Log"
msgstr "Изтриване на дневника"

#: main.py:178
msgid "Select Image"
msgstr "Избор на образ"

#: main.py:179 main.py:289 main.py:308
msgid "No Image Selected"
msgstr "Не е избран образ"

#: main.py:181 main.py:248
msgid "No Checksum Generated"
msgstr "Не е генерирана контролна сума"

#: main.py:183 main.py:355
msgid "No Mount Point Selected"
msgstr "Не е избрана точка за монтиране"

#: main.py:185
msgid "Mounted Images:"
msgstr "Монтирани образи:"

#: main.py:186
msgid "Drag image files into above window to quickly mount"
msgstr "Плъзнете файлове-образи в горния прозорец за бързо монтиране"

#: main.py:196
msgid "Mount Point"
msgstr "Точка на монтиране"

#: main.py:197
msgid "Image File"
msgstr "Образ"

#: main.py:236
msgid "_Cancel"
msgstr "_Отказ"

#: main.py:240
msgid "Checksum generation started..."
msgstr "Генерирането на контролната сума е стартирано..."

#: main.py:256
#, python-format
msgid "%s:%s"
msgstr "%s:%s"

#: main.py:259
msgid "Checksum generation aborted!"
msgstr "Генерирането на контролната сума е прекъснато!"

#: main.py:268
msgid "Open Image.."
msgstr "Отваряне на образ.."

#: main.py:288
msgid ""
"File not found. \n"
"Please check source exists."
msgstr ""
"Файлът не е открит. \n"
"Моля проверете дали източникът съществува."

#: main.py:307
msgid ""
"File does not appear to be a compatible Image. \n"
"Please check source."
msgstr ""
"Изглежда, че файлът не е съвместим образ.\n"
"Моля проверете източника."

#: main.py:318
#, python-format
msgid "%s not found."
msgstr "%s не е открит."

#: main.py:322
msgid "Are you sure you wish to delete the log?"
msgstr "Сигурни ли сте, че желаете да изтриете дневника?"

#: main.py:326 main.py:328
#, python-format
msgid ""
"Error deleting log.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при изтриване на дневника.\n"
"Грешка на операционната система(%s): %s"

#: main.py:330
#, python-format
msgid ""
"Error deleting log.\n"
"Unexpected error: %s"
msgstr ""
"Грешка при изтриване на дневника.\n"
"Неочаквана грешка: %s"

#: main.py:393
msgid "Application closed!"
msgstr "Приложението е затворено!"

#: main.py:397
#, python-format
msgid ""
"%s is still mounted.\n"
"Do you wish to unmount it before exiting?"
msgstr ""
"%s все още е монтиран.\n"
"Желаете ли да го демонтирате преди изход?"

#: main.py:408
#, python-format
msgid ""
"Error saving unmounted images to file.\n"
"OS error(%s): %s"
msgstr ""
"Грешка при запазаване на демонтираните образи във файл.\n"
"Грешка на операционната система(%s): %s"

#: main.py:410
#, python-format
msgid ""
"Error saving unmounted images to file\n"
"Unexpected error: %s"
msgstr ""
"Грешка при запазаване на демонтираните образи във файл.\n"
"Неочаквана грешка: %s"
